tests/system/action/motion/test_create.py

```diff
@@ -175,87 +175,6 @@ def test_create_missing_state(self) -> None:
             response.json["message"]
         )
 
-    def test_correct_origin_id_set(self) -> None:
```

Comment:
Add a small test with an `origin_id` set and assert that is rejected due to the `origin_id`.

Reply:
Add a missing origin test. It is a short test, where the origin_id is set and the motion is missing.

Reply:
Is that you are asking for? If not, please explain in more detail the test, which is missing.

Reply:
Yes, this sounds good. But where is this test?

Reply:
The test should be `test_missing_origin`